src/hostname/hostnamectl.c

```diff
@@ -198,6 +216,19 @@ static int print_status_info(StatusInfo *i) {
                         return table_log_add_error(r);
         }
 
+        if (i->os_support_end != USEC_INFINITY) {
+                usec_t n = now(CLOCK_REALTIME);
+
+                r = table_add_many(table,
+                                   TABLE_FIELD, "OS Support End",
+                                   TABLE_TIMESTAMP_DATE, i->os_support_end,
+                                   TABLE_FIELD, n < i->os_support_end ? "Support Left" : "Over Support",
```

Comment:
I'd prefer "End of Life" or "End of Support/Support Ended"

Reply:
note that we output the field twice here: once as absolute date, and once as relative time. 

The former field is called "OS Support End". it would be very confusing if the latter would be called "End of Support" too. I think we need to emphasize a bit the fact that the second field is relative. Hence "Support Left", and "Over Support".

Reply:
I see. I don't think there is value in telling the user how much time they've spent without support. It imho makes more sense to have this be "Support Left" and just say "End of Life/Support" if we're over it (or just 0 if the table system doesn't do that).

Reply:
I mean, I am not saying that "Over Support" is great English. Open of ideas. But I still think we should show this info, a system 4y out of support is *much* more problematic than one which is 2 weeks out of support after all, and hence displaying that explicitly makes sense I am sure (as opposed to just a boolean)

Reply:
If you ask me, 2 weeks EOL requires just as much attention as 4 years…

Would be helpful to see example output. But "Unsupported for"/"Without Support (for)" could work.

Reply:
Example:

```
 Static hostname: zeta
       Icon name: computer-laptop
         Chassis: laptop 💻
      Machine ID: e1698c0a8e1b4654b76e0fa3cb135882
         Boot ID: 0808e0755fb142c7a4914499db3187f0
Operating System: Fedora Linux 37 (Workstation Edition)
     CPE OS Name: cpe:/o:fedoraproject:fedora:37
  OS Support End: Tue 2023-11-14
    Support Left: 9month 2w 6d
          Kernel: Linux 6.1.6-200.fc37.x86_64
    Architecture: x86-64
 Hardware Vendor: Lenovo
  Hardware Model: ThinkPad X1 Carbon Gen 9
Firmware Version: N32ET82W (1.58 )
   Firmware Date: Mon 2022-12-05
```

Reply:
Mind you, as a fellow german I can understand you using "over", but I am pretty sure this is bad english.

Reply:
maybe "Out of OS Support For:"?

@keszybz is usually pretty good at this, what do you say?

Reply:
btw, the new output did work on me: it made an impression on me to realize that currentl stable fedora is only support for 9 more month...

Reply:
> btw, the new output did work on me: it made an impression on me to realize that currentl stable fedora is only support for 9 more month...

I am not doubting the value if it's still supported. I am merely saying that once it's EOL, there's no point in counting (and showing) the days.

Reply:
Just thought of another one: "OS Support Remaining/Expired"

Reply:
changed